### PR TITLE
Add org.kde.ruqola

### PR DESCRIPTION
**Pull request by nicolasfella**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by nicolasfella** [comment]
bot, build org.kde.ruqola

**Comment by nicolasfella** [comment]
bot, built org.kde.ruqola

**Comment by nicolasfella** [comment]
bot, build org.kde.ruqola

**Comment by nicolasfella** [comment]
bot, build org.kde.ruqola

File: org.kde.ruqola.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+    "id": "org.kde.ruqola",
+    "branch": "master",
```
Comment (hfiguiere) [inline org.kde.ruqola.json:3]:
```suggestion
```

File: org.kde.ruqola.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "ruqola",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
```
Comment (hfiguiere) [inline org.kde.ruqola.json:12]: that's a lot of permissions for a chat client.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
it can't find the 128x128 icon.

File: org.kde.ruqola.json (hunk 2 above)
Comment (nedrichards) [reply org.kde.ruqola.json:12]: Yes, what do you need to read or write that isn't handled by the portals already?

**Comment by nicolasfella** [comment]
bot, build org.kde.ruqola

**Comment by nicolasfella** [comment]
>FB: Running: patch -p1 -i /srv/buildbot/worker/build-x86_64-6/build/0003-add-128x128-version-of-the-icon.patch
File src/icons/128-apps-ruqola.png: git binary diffs are not supported.

Hmm, I guess this will have to wait for the next tarball release

**Comment by hfiguiere** [comment]
> > FB: Running: patch -p1 -i /srv/buildbot/worker/build-x86_64-6/build/0003-add-128x128-version-of-the-icon.patch
> > File src/icons/128-apps-ruqola.png: git binary diffs are not supported.
> 
> Hmm, I guess this will have to wait for the next tarball release

you could fetch it directly from the repository.

**Comment by nicolasfella** [comment]
bot, build org.kde.ruqola

**Comment by barthalion** [comment]
@flathub/kde, please approve

**Review by CarlSchwan — APPROVED** [review]

**Pull request closed without merging**